packages/browser/src/plugins/segmentio/batched-dispatcher.ts

```diff
@@ -23,6 +24,15 @@ function approachingTrackingAPILimit(buffer: unknown): boolean {
   return kilobytes(buffer) >= MAX_PAYLOAD_SIZE - 50
 }
 
+/**
+ * Checks if payload is over the limit for keepalive
+ * requests. If keepalive is enabled we want to avoid
+ * going over this to prevent data loss.
+ */
+function passedKeepaliveLimit(buffer: unknown): boolean {
+  return kilobytes(buffer) >= 64
```

Comment:
Should we add padding, since in practice the max size is the total of all outbound requests with keep-alive, so we are potentially competing with other requests?